configs/trainers/audio_jepa/bool_mask.yaml

```
@@ -0,0 +1,35 @@
+_target_: ami.trainers.bool_mask_audio_jepa_trainer.BoolMaskAudioJEPATrainer
+
+partial_dataloader:
+  _target_: torch.utils.data.DataLoader
+  _partial_: true
+  batch_size: 8
+  shuffle: true
+  drop_last: true # for batchnorm.
+  # based on the original paper settings
+  collate_fn:
+    _target_: ami.trainers.components.bool_audio_jepa_mask_collator.BoolAudioJEPAMultiBlockMaskCollator
+    input_sample_size: ${models.audio_jepa_target_encoder.model.input_sample_size}
+    patch_sample_size: 400
+    stride: 320
+    mask_scale: [0.1, 0.25]
+    n_masks: 4
+    min_keep: 10
```

パッチ数的に4くらいが良いだろう(10%)
```suggestion
    min_keep: 4
```

---

パッチ数は50だから、10%にしたいなら`min_keep: 5`の方がいいかと思った　ということで修正
https://github.com/MLShukai/ami/pull/267/commits/f12b9810573d5dc7730e5ab6e0ceae2c0943d955